README.txt

```diff
@@ -3,38 +3,46 @@ import os
 import numpy as np
 import shutil
 
+#annotate_img("B.jpg","MS",10,120)
+
+global COLOR
```

Comment:
Mods in `README.txt` by accident again?

Reply:
Sorry about that, I updated it.

Reply:
Updated README with Markdown formatting.